server/server.js

```diff
@@ -43,6 +43,7 @@ app.get('/booking', async (req, res) => {
           res.status(201).send(err);
         });
     })
+    //if call to Overview fails, initialize Booking Service with campId: 0
     .catch((err) => {
       db.Booking.find({campId: 0})
         .then((response) => {
```

Comment:
again, if we only care about something specific, then destructure it so the rest of the noise is filtered out

Reply:
in this case you would have a `.then(([site]) => ... )` block